Makefile

```diff
@@ -22,6 +22,10 @@ create_db:
 lint:
 	pre-commit run --all-files
 
+.PHONY: test
+test:
+	pytest -vsx ./prijateli_tree/tests/
+
```

Comment:
Added a `make test` command so that you can run all tests within that folder. It will fail if any of the tests in it fail.

Reply:
cc: @FedericoDM